src/add-task.test.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable prefer-const */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Great job so far @NewtonOmbese and @tan12082001 There are just a few changes I will suggest you implement and you'll be on your way to the next milestone :+1: 
- [ ] I noticed that you used a lot of `eslint disable` commands in your project. I recommend avoiding the use of `eslint disable commands` in your projects, as these errors and warnings are typically helpful for identifying areas where you can improve your code. By resolving these issues, you can write more clean and readable code that adheres to industry standards, which can make it easier to maintain your codebase over time. :100:
- [ ] So kindly remove all the `eslint disable` commands in your project :+1: